tensorflow_datasets/core/features/video_feature.py

```diff
@@ -61,6 +62,8 @@ def __init__(self, shape):
       raise ValueError('Video shape should be of rank 4')
     if shape.count(None) > 1:
       raise ValueError('Video shape cannot have more than 1 unknown dim')
+    if shape[-1] not in (1, 3):
```

Comment:
Please remove. The test is already in Image() called bellow: https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/core/features/image_feature.py#L108

Reply:
done, but does this mean videos with 0 or 2 channels are also accepted? In the interest of keeping documentation up to date (had me confused initially when it said channels had to be 3).